examples/notebooks/beam-ml/per_key_models.ipynb

```diff
@@ -107,12 +107,7 @@
         }
       ],
       "source": [
-        "# Note that this notebook currently installs from Beam head since this feature hasn't been released yet.\n",
-        "# It will be released with version 2.51.0, at which point you can install with the following command:\n",
-        "# !pip install apache_beam[gcp]>=2.51.0 --quiet\n",
-        "!git clone https://github.com/apache/beam\n",
-        "!pip install -r beam/sdks/python/build-requirements.txt\n",
-        "!pip install -e ./beam/sdks/python[gcp]\n",
+        "!pip install apache_beam[gcp]>=2.51.0 --quiet\n",
```

Comment:
@AnandInguva @damccorm I'm planning to merge this after the 2.51.0 release, so this section needs to be updated, but please check to see if I updated it correctly. Thank you!

Reply:
Ah, I actually had a pr open to do this already (and it is now merged - #28801)